app/views/external_users/claims/case_details/_fields.html.haml

```diff
@@ -16,11 +16,16 @@
 
     = f.collection_select :court_id, Court.alphabetical, :id, :name, { include_blank: ''.html_safe }, { class: 'form-control fx-autocomplete', id: 'court', 'aria-label': t('.court') }
 
-
-  = f.adp_text_field :case_number,
-    label: t('.case_number'),
-    hint_text: t('.case_number_hint'),
-    errors: @error_presenter
+  - if Settings.urn_enabled?
```

Comment:
[see above](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3457/files#r472029346)

Reply:
👍 thank you for the suggestion, I have amended to use dynamic translations (option 2).